src/west/configuration.py

```diff
@@ -244,6 +244,11 @@ def _location(cfg: ConfigFile, topdir: Optional[PathType] = None) -> str:
             pd = PureWindowsPath(os.environ['ProgramData'])
             return os.fspath(pd / 'west' / 'config')
 
+        if 'MSYS_NT' in plat:
+            # MSYS2 is supported like Cygwin
```

Comment:
OK, but then please write this as follows to avoid duplication:

```py
if 'CYGWIN' in plat or 'MSYS_NT' in plat:
```

Reply:
I made the suggested change, and force-pushed.